package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "focus-app",
+  "version": "1.0.0",
+  "description": "Web app that allows you to play various sounds to help users focus ",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
i see that we've adding some new dev dependencies but haven't added any NPM scripts yet. 

If you're running stuff like babel as of now, what commands/etc are you using to get it done? 

Maybe we can move those into NPM scripts for ease of use and for other devs to onboard more quickly :)